core/modules/image/image.install

```diff
@@ -90,14 +90,14 @@ function image_requirements($phase) {
       }
       else {
         $requirements['image_gd']['severity'] = REQUIREMENT_WARNING;
-        $requirements['image_gd']['description'] = t('The GD Library for PHP is enabled, but was compiled without support for functions used by the rotate and desaturate effects. It was probably compiled using the official GD libraries from http://www.libgd.org instead of the GD library bundled with PHP. You should recompile PHP --with-gd using the bundled GD library. See <a href="http://www.php.net/manual/book.image.php">the PHP manual</a>.');
+        $requirements['image_gd']['description'] = t('The GD Library for PHP is enabled, but was compiled without support for functions used by the rotate and desaturate effects. It was probably compiled using the official GD libraries from http://www.libgd.org instead of the GD library bundled with PHP. You should recompile PHP --with-gd using the bundled GD library. See <a href="https://www.php.net/manual/book.image.php">the PHP manual</a>.');
       }
     }
     else {
       $requirements['image_gd'] = array(
         'value' => t('Not installed'),
         'severity' => REQUIREMENT_ERROR,
-        'description' => t('The GD library for PHP is missing or outdated. Check the <a href="@url">PHP image documentation</a> for information on how to correct this.', array('@url' => 'http://www.php.net/manual/book.image.php')),
+        'description' => t('The GD library for PHP is missing or outdated. Check the <a href="@url">PHP image documentation</a> for information on how to correct this.', array('@url' => 'https://www.php.net/manual/book.image.php')),
```

Comment:
Line breaks... :smile_cat: I know, this might get frustrating, so thanks again for your effort and patience.

Reply:
Essentially, you are suggesting to change where an array definition starts in every line where I changed a php.net url, aren't you?

Reply:
Yes, essentially that. :+1: 

The automated action fails to annotate, but I'd like you to fix all coding standard issues in your PR - in code you touched, so we can see, if that makes the check pass.